Update filtering.mdx (#3491)
grommet/hpe-design-system: aries-site/src/pages/templates/filtering.mdx

  becomes limited, use a single filter button that opens a Layer containing the filters.
- **Is there more than one filter?** Use a single button with filter icon that
  [opens a Layer containing the filters](/templates/filtering).
- **When only one item can be selected from a list, use the [Select](/components/select) component.
- **Is the user going to be selecting from 4 or fewer multiple selection options?** Filter with
  [CheckBoxGroup](#filters-in-a-center-layer).

> - **If the filter set requires multiple selections can be made in a list that contains 5 or more items, use [SelectMultiple](/components/SelectMultiple).

Reviewer: ```suggestion
- **If the filter set allows for multiple selections and there are 5 or more items to choose from, use [SelectMultiple](/components/SelectMultiple).
```

> - **When the data list contains 10 or more items, consider including search in your [SelectMultiple](/components/SelectMultiple) implementation.

Reviewer: ```suggestion
- **When the data contains 10 or more items, consider including search in your [SelectMultiple](/components/SelectMultiple) implementation.
```
